docs/concepts.rst

```diff
@@ -842,61 +842,124 @@ values by "case" instead of by entity.
-computation, as Bionic may need to compute that entity's dependencies, and their
-dependencies, and so on. By default, Bionic computes these values one at a time.
-However, it can also be configured to compute them in parallel; depending on the
-structure of your flow, this can be significantly faster.
-
-Parallel execution can be enabled like this:
+Bionic can use multiple CPU cores to compute entities in parallel. This feature
+is useful if you have many expensive operations which do not depend on each
+other.
 
 .. code-block:: python
 
     builder.set("core__parallel_execution__enabled", True)
 
 When parallel execution is enabled, Bionic starts up several worker processes
-[#workers]_, each of which can work on one value at a time. Of course, a worker can
-only start computing a value once all its dependencies are complete, so the number of
-processes that can be working at once depends on the :ref:`dependency graph
-<dagviz>`: if there aren't many branches in the graph, then most of the processes
-won't do much work. It does take extra time to set up the processes and move
-information between them, so parallel execution is not guaranteed to be faster
-overall. However, in general, if you have many expensive operations which don't
-depend on each other, enabling parallelism will improve performance.
-
-By default, Bionic will create one worker process for each CPU on your machine. This
-is usually a sensible number, but it can also be set directly:
+[#workers]_, each of which can work on one value at a time. By default, Bionic
+will create one worker process for each CPU on your machine. This is usually a
+sensible number, but it can also be set directly:
 
 .. code-block:: python
 
     builder.set("core__parallel_execution__worker_count", 8)
 
+.. [#workers] The pool of workers is managed by
+  `Loky <https://loky.readthedocs.io/en/stable/>`_,
+  which is built on Python's
+  `multiprocessing <https://docs.python.org/3.8/library/multiprocessing.html>`_ module.
+  The pool is global and reusable, so it should only need to be initialized once in
+  the lifetime of the main process.
+
+.. _google-aip:
+
+Google AI Platform Execution
+............................
+
+.. versionadded:: 0.9.0
+
+Bionic can use compute resources in the cloud by sending entities to
+`Google AI Platform (AIP) <https://cloud.google.com/ai-platform>`_ for
+computation. The GCP project needs to be specified.
+
+.. code-block:: python
+
+    builder.set('core__aip_execution__enabled', True)
+    builder.set('core__aip_execution__gcp_project_name', 'my-gcp-project')
+
+This feature requires having a Docker image in Google Container Registry
+containing the same Python environment (runtime and libraries) as the one used
+in the local environment Bionic is running in.
+
+Bionic can build this Docker image if all the Python libraries in the local
+environment are installed through ``pip``. Otherwise, you will need to specify the
+Docker image.
```

Comment:
s/./:/

Reply:
I think you meant the preceding full stop?

Reply:
I meant it for the period at the end of "specify the Docker image.", because the code below is a demonstration of this. A colon makes the relationship between the code and the text clearer.